scripts/release/fetch-latest-yaml.sh

```diff
@@ -59,7 +59,7 @@ fetch_engine_operator_yaml()
 
   sed -e 's@\:ci@'"\:$TAG"'@' -i ${TEMP_RESP_FILE}
   sed -e 's@\: ci@'"\: $TAG"'@' -i ${TEMP_RESP_FILE}
-  sed -e 's@\: dev@'"\: $TAG"'@' -i ${TEMP_RESP_FILE}
+  sed -e 's@\: dev$@'"\: $TAG"'@' -i ${TEMP_RESP_FILE}
```

Comment:
what this `$` meant for here?

Reply:
it is the syntax of regular expression that the word should end after `dev`. Without that, the `device` will also get matched.